e/elasticsearch/Dockerfiles/template/Dockerfile

```diff
@@ -0,0 +1,155 @@
+################################################################################
+# This Dockerfile was generated from the template at distribution/src/docker/Dockerfile
+#
+# Beginning of multi stage Dockerfile
+################################################################################
+
+################################################################################
+# Build stage 0 `builder`:
+# Extract Elasticsearch artifact
+################################################################################
+
+FROM registry.access.redhat.com/ubi8/ubi:latest AS builder
+
+ARG PATCHFILE="https://raw.githubusercontent.com/ppc64le/build-scripts/master/e/elasticsearch/elasticsearch_8.1.1.patch"
```

Comment:
Either Use `PATCH` instead of PATCHFILE, as we are using "PATCH" in our config file (build_info.json), or use `PATCHFILE` as the argument in the config file.